wordpress-mobile/WordPress-Android
# Use site timezone in the Overview card

The FluxC changes need to be merged first - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1323

This PR adds a "Site timezone" text to the date selector whenever the site timezone is different that the current timezone. The text is always in `UTC + 5:30` format. You can set the site timezone in wordpress.com settings. 

I've extracted the common UI code for the date selector into a separate file.

This PR also contains the FluxC changes for loading data in the overview card.

<img width="486" alt="Screenshot 2019-07-30 at 10 46 55" src="https://user-images.githubusercontent.com/1079756/62124177-d5ffea00-b2c9-11e9-8f72-a1bb605947fe.png">
<img width="486" alt="Screenshot 2019-07-30 at 10 46 36" src="https://user-images.githubusercontent.com/1079756/62124178-d5ffea00-b2c9-11e9-8176-ed67ad77eff0.png">


To test:
* Change timezone of your site 
* Go to the app
* Check that the Date selector now shows the `Site timezone (UTC + X)` subtitle
* Change the timezone back to your current timezone
* The Date selector doesn't show the Site timezone

To test 2: 
* Change timezone of your site to UTC + 14/-14 (it could be different, but it's the timezone that moves your current date)
* Go to the app
* Reload the stats (if necessary)
* Check that the max date in the Overview card on the Days tab is different than your current date in the app

Update release notes:
